Library/Homebrew/vendor/ruby/opt

```diff
@@ -1 +0,0 @@
-2.0.0-p648
```

Comment:
I might not understand the full context, but it feels wrong that this is added in a previous commit and then removed in this commit. Is that intentional?